### ISSUE TITLE
Communicate security expectations

### ISSUE THREAD

**Issue by kcole16** [nearprotocol/near-wallet#442]
## Overview

It's important to clearly communicate the security expectations of the NEAR Wallet to users. 

This should include: 
- Some indication of a "safe" amount of assets to store in the Wallet
- Recommendations for the recovery options
- Information re: more secure options (third party wallets, hardware wallets, etc)

## Story

As a user of the NEAR wallet, I'd like to understand how much NEAR/DAI/ETH/CryptoKitties I can should store before using a mobile wallet or purchasing a hardware wallet.

**Comment by kcole16** [comment]
@Patrick1904 @vgrichina @corwinharrell Thoughts?

**Comment by corwinharrell** [comment]
@kendall i think this is a good start. One thing I would want to make sure that we are considering is that when we are communicating around storage of larger amounts, we don't want to "talk ourselves down" in our messaging in the sense that I don't think we want to necessarily frame ourselves as "less secure" than other wallets. We should strategize about how to encourage users to keep larger sums in other wallets without using language that is degrading towards NEAR (degrading may be the wrong word but I think you get my point.)

**Comment by Patrick1904** [comment]
> @kendall i think this is a good start. One thing I would want to make sure that we are considering is that when we are communicating around storage of larger amounts, we don't want to "talk ourselves down" in our messaging in the sense that I don't think we want to necessarily frame ourselves as "less secure" than other wallets. We should strategize about how to encourage users to keep larger sums in other wallets without using language that is degrading towards NEAR (degrading may be the wrong word but I think you get my point.)

Good point @corwinharrell. Copy will play an important role here. I think we'll want to communicate something like while web wallets enable seamless interactions with web applications, they currently lack some security features that hardware or native application wallets can provide. We therefore don’t recommend that you store large amounts of NEAR in your web wallet to minimize the chance of significant loss should your account ever become compromised.

**Comment by kcole16** [comment]
Note: Explore language re: privacy considerations (all activity is public) cc @corwinharrell @vgrichina
